Lesson_3/task_1.py

```diff
@@ -0,0 +1,4 @@
+i = 0
+while i < 42:
```

Comment:
You shouldn't use loops here. This task can be solved using arithmetical operators, like *